### PR TITLE
Update nginx config

### PR DESCRIPTION
**Pull request by kulga**
Resolves https://github.com/flarum/core/issues/955
#955

### CODE REVIEW

File: _docs/installation.md [hunk 1]
```diff
@@ -63,6 +63,14 @@ Flarum includes a `.htaccess` file – make sure it's been uploaded correctly.
 Add the following lines to your server's configuration block:
```
Comment (franzliedke) [inline _docs/installation.md:63]: This line would be changed...

**Comment by franzliedke** [comment]
I'm not 100% sure about this.

We would probably need some more explanation / details about which parts are needed when, and what they mean.

**Comment by tobyzerner** [comment]
Yep, we were discussing this in the meeting. I think the conclusion we came to was that we should add some comments next to the placeholder lines that need to be changed.

And should we change the intro line to something like "Your nginx server configuration should look something like this:" ?

**Comment by kulga** [comment]
I was thinking a few ways to do this
- inline
- a more elaborate multi-line explanation
- examples and explanation
- examples and referential links

Eg

```
    listen 80; # HTTP, use 443 ssl for HTTPS. Can also be 1.2.3.4:80
```

```
# listen defines the port used. 80 is used for http
# 443 and ssl are used for listening to https connections
# and require certificate configuration to work properly
listen 80;
```

```
# listen defines the port used. 80 is used for http
# 443 and ssl are used for listening to https connections
# and require certificate configuration to work properly
# If using ssl uncomment the following line

# listen 443 ssl;
listen 80;
```

```
# listen defines the port used. 80 is used for http
# See this link for more details
# http://nginx.org/en/docs/http/ngx_http_core_module.html#listen

listen 80;
# listen 443 ssl;
```

**Comment by franzliedke** [comment]
I vote inline.

IMO, a big problem with Nginx configs in the internet are mostly trying to copy some stuff together until it works, without knowing why or how to do it better. (Note, I don't exclude myself from that.) That probably also says about the power / usability of these configs...

**Comment by luceos** [comment]
I think inline will not give us the opportunity to explain what it does sufficiently. I like the multi line solution better..

**Comment by franzliedke** [comment]
@kulga Can you adapt this accordingly?

**Comment by tobyzerner** [comment]
@kulga bump :) are you able to do this still?

**Comment by franzliedke** [comment]
@kulga May I nudge you again? Otherwise, anybody else interested in taking this over?

**Comment by kulga** [comment]
I put some commentary in.
I also included a part for HTTPS encryption.

Unfortunately, I'm shorter on time now. It may well be better to have someone that can dedicate more time to this complete this :)

**Comment by tobyzerner** [comment]
I don't know if we should have config for SSL in here. I feel like it's outside the scope of what we need to provide. Perhaps a link to a good page/guide would suffice.

**Comment by franzliedke** [comment]
@tobscure Agreed. The more we provide, the more we need to keep up-to-date / in sync for the different server platforms.

**Comment by tobyzerner** [comment]
@franzliedke I think this can be closed actually. We've removed non-Flarum-specific config from the nginx.conf we distribute in flarum/flarum. It's the user's responsibility to do the rest of their config, we don't need to document it.

**Pull request closed without merging**